cpg_workflows/stages/mito.py

```diff
@@ -259,3 +262,221 @@ def queue_jobs(
+        main = sequencing_group.dataset.prefix()
+        analysis = sequencing_group.dataset.analysis_prefix()
+        return {
+            'out_vcf': main / 'mito' / f'{sequencing_group.id}.mito.vcf.bgz',
+            'haplocheck_metrics': analysis
+            / 'mito'
+            / f'{sequencing_group.id}.haplocheck.txt',
+        }
+
+    def queue_jobs(
+        self, sequencing_group: SequencingGroup, inputs: StageInput
+    ) -> StageOutput | None:
+        # Mitochondrial specific reference files.
+        mito_ref = get_batch().read_input_group(**MITO_REF)
+        shifted_mito_ref = get_batch().read_input_group(**SHIFTED_MITO_REF)
+
+        jobs = []
+
+        # Get input resources
+        non_shifted_cram = get_batch().read_input_group(
+            cram=str(inputs.as_path(sequencing_group, RealignMito, 'non_shifted_cram')),
+            crai=str(inputs.as_path(sequencing_group, RealignMito, 'non_shifted_cram'))
+            + '.crai',
+        )
+        shifted_cram = get_batch().read_input_group(
+            cram=str(inputs.as_path(sequencing_group, RealignMito, 'shifted_cram')),
+            crai=str(inputs.as_path(sequencing_group, RealignMito, 'shifted_cram'))
+            + '.crai',
+        )
+        if get_config()['mito_snv']['use_verifybamid']:
+            verifybamid_output = get_batch().read_input(
+                str(inputs.as_path(sequencing_group, CramQC, 'verify_bamid')),
+            )
+        else:
+            verifybamid_output = None
+
+        # Call variants on WT genome
+        call_j = mito.mito_mutect2(
+            b=get_batch(),
+            cram=non_shifted_cram,
+            reference=mito_ref,
+            region='chrM:576-16024',  # Exclude the control region.
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(call_j)
+        assert isinstance(call_j.output_vcf, hb.ResourceGroup)
+
+        # Call variants in ONLY the control region using the shifted reference
+        shifted_call_j = mito.mito_mutect2(
+            b=get_batch(),
+            cram=shifted_cram,
+            reference=shifted_mito_ref,
+            region='chrM:8025-9144',  # Only call inside the control region.
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(shifted_call_j)
+        assert isinstance(shifted_call_j.output_vcf, hb.ResourceGroup)
+
+        # Merge the wt and shifted VCFs
+        merge_j = mito.liftover_and_combine_vcfs(
+            b=get_batch(),
+            vcf=call_j.output_vcf,
+            shifted_vcf=shifted_call_j.output_vcf,
+            reference=mito_ref,
+            shift_back_chain=shifted_mito_ref.shift_back_chain,
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(merge_j)
+        assert isinstance(merge_j.output_vcf, hb.ResourceGroup)
+
+        # Merge the mutect stats output files (needed for filtering)
+        merge_stats_J = mito.merge_mutect_stats(
```

Comment:
nit? not sure if intentional 
```suggestion
        merge_stats_j = mito.merge_mutect_stats(
```

Reply:
Nope, just poor consistency on my part 😅 I will change them.